public/stylesheets/scss/_variables.scss

```diff
@@ -0,0 +1,5 @@
+$font-light: #B4C4CF;
```

Comment:
For a product it's better to know the element that's being affected. I mean, if tomorrow that blue changes to pink, the name of the variable should also change. Variables are helpful to keep changes through time simple, today blue, tomorrow pink but always the same variable. Don't know if I was clear :) Let me know what you think @pletcher